bin/install-wp-tests.sh

```diff
@@ -166,6 +172,11 @@ install_deps() {
 	cd "$WP_CORE_DIR"
 
 	curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+
+	# delete existing wp-config and woocommerce repository
+	rm wp-config.php || echo "wp-config.php does not exist"
```

Comment:
`rm -f` could be used here to hide the command’s error message when the file does not exist. I see this in the CI logs:

```
+rm wp-config.php
rm: cannot remove ‘wp-config.php’: No such file or directory
+echo 'wp-config.php does not exist'
```

Reply:
Good point!